catalog/spatial/registry/registry-policy-plugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -17,26 +17,41 @@
          name="Registry Policy Plugin"
          id="org.codice.ddf.registry.policy.RegistryPolicyPlugin">
 
-        <AD description="Roles/attributes required for CUD (create/update/delete) operations on registry entries. Example: {role=role1;type=type1}"
-            name="Registry CUD Attributes" id="writeAccessPolicyStrings" required="true" type="String" cardinality="100"
+        <AD description="Roles/attributes required for create operations. Example: {role=role1;type=type1}"
```

Comment:
Do you want to be consistent and add "...on registry entries. to this description as in the update and delete descriptions?